### PR TITLE
Enhanced user status information

### PR DESCRIPTION
**Pull request by Syrgak-Alan**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [✖️]
| New feature/enhancement? (use the a.x branch)      | [✔️]
| Deprecations?                          | [✖️]
| BC breaks? (use the c.x branch)        | [✖️]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
Leuchtfeuer.com

This PR itroduced better visibility of the Mautic users status:
* Show "Last Login" and "Publication status" in the Users list table
* Show "Recent Acivity " in the User detail view

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to "Users" list view - you should see the new columns  "Last Login" and "Publication status"
3. When you create a new user, log in, deactivate, ... that show up here
4. When you open one users details, you should see recent activities on the right hand side.

### CODE REVIEW

File: app/bundles/UserBundle/Resources/views/User/list.html.twig
Comment (kuzmany) [inline app/bundles/UserBundle/Resources/views/User/list.html.twig]: I like the PR and works as expected. 
But the status should be translateable.
You can add a translation key or you can use the same switch button before the name like we use in other entities (Email etc...)

https://github.com/mautic/mautic/blob/d71760b47dc9ba0add89e26099db025c4d2279b2/app/bundles/EmailBundle/Resources/views/Email/_list.html.twig#L116

image

**Review by kuzmany — CHANGES_REQUESTED** [review]

File: app/bundles/UserBundle/Resources/views/User/list.html.twig
Comment (Syrgak-Alan) [reply app/bundles/UserBundle/Resources/views/User/list.html.twig]: Hi! I added switch button before the name as You said, check it please

**Review by kuzmany — DISMISSED** [review]
Good to me 👍

**Comment by Syrgak-Alan** [comment]
@kuzmany Hi, now i need to add some phpUnit test, right?

**Review by Syrgak-Alan — COMMENTED** [review]
waiting for approve

**Comment by kuzmany** [comment]
@Syrgak-Alan Yes, need to fix phpstan issues first https://github.com/mautic/mautic/actions/runs/6330404563/job/17242278771?pr=12711 and then would be great to write tests. I think the unit test for display detail of the user page and check If any logs exist there. 

You can lil bit inspiration here https://github.com/mautic/mautic/blob/90253cd28bdb65e112e3875973b712fa9f97def5/app/bundles/UserBundle/Tests/Functional/Controller/RoleControllerFunctionalTest.php

**Review by PatrickJenkner — APPROVED** [review]
Works as described. Thanks 👍

**Comment by RCheesley** [comment]
Looks good folks, let's get the PHPSTAN error fixed and code coverage up then we should be able to merge once it's rebased to the 5.1 branch! 🚀

**Comment by andersonjeccel** [comment]
Look awesome. 
In terms of UX/UI, we should add a new translation string to make it clear that it's a user deactivation/activation instead of published/unpublished.
Aside from the Codecov issue, it's a very useful addition.

**Review by andersonjeccel — COMMENTED** [review]
Looks awesome.
In terms of UX/UI, we should add a new translation string to make it clear that it's a user deactivation/activation instead of published/unpublished.
Aside from the Codecov and other code issues, it's a very useful addition.

**Comment by RCheesley** [comment]
@Syrgak-Alan we can't move forward with this until you fix the failing tests. Could you take a look please? Many thanks!

**Comment by Syrgak-Alan** [comment]
@RCheesley sorry, I didn't see before. Gonna do it right now

**Comment by Syrgak-Alan** [comment]
@RCheesley can I create new branch and create new PR where the problems gonna be solved?

**Comment by RCheesley** [comment]
If you can rebase to 5.1 and fix the issues on this PR that would be better. There should hopefully not be too many problems with rebasing at this point in time. But if it's causing too many conflicts then sure, make a new PR. We will need to re-test when it's rebased anyway.

**Comment by Syrgak-Alan** [comment]
@RCheesley Hi! I tried to rebase this PR to 5.1, but I couldn't
Now as I said I want to create another one, which will be PR to 5.1 branch, right?

**Comment by Syrgak-Alan** [comment]
@RCheesley Hi, I want to create another PR. Should I push it to 5.x?

**Comment by RCheesley** [comment]
Please use the 5.1 branch as your base, as there won't be any more feature releases on the 5.x branch now. 5.1 is the next feature-based release.

**Comment by Syrgak-Alan** [comment]
@RCheesley I was asking, because in 5.1 branch there appears an issue from UserBundle/Model/PasswordStrengthEstimatorModel.php

**Comment by RCheesley** [comment]
Please chat in #t-product if you require assistance with the PR - the core team will be able to answer any questions.

**Comment by Syrgak-Alan** [comment]
@RCheesley Hi! As I said, created new PR with base 5.1, here is number of PR: 13074
If I did everything correct, please tell me, so I could close this PR

**Comment by kuzmany** [comment]
Close favor to https://github.com/mautic/mautic/pull/13074

**Pull request closed without merging**
